spec/services/process_notify_callback_spec.rb

```diff
@@ -51,6 +51,16 @@
 
         expect(process_notify_callback).to eq(:not_found)
       end
+
+      it 'send a new referee request email to the candidate' do
+        allow(SendNewRefereeRequestEmail).to receive(:call)
+          .with(application_form: application_form, reference: reference, reason: :email_bounced)
```

Comment:
I wonder if we want more of an integration test for this (in system or request specs), to avoid testing the internals of the service here.

Reply:
Makes sense! Done in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1157/commits/70bb3012837a2bb8368b9f3656acf6afcea9414b.